.github/workflows/create_changeset_pr_for_workspace.yml

```diff
@@ -0,0 +1,65 @@
+name: Create Changeset PR for workspace
+
+on:
+  workflow_dispatch:
+    inputs:
+      workspace:
+        description: 'Name of the Workspace'
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      workspace:
+        description: 'Name of the Workspace'
+        required: true
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ inputs.workspace }}
+
+jobs:
+  changesets-pr:
+    name: Update Version Packages PR for ${{ inputs.workspace }} on branch main
```

Comment:
Is it possible that we run this workflow on branches besides `main`?

Reply:
not at the moment

Reply:
yeah it makes sense, I can add the feature. I've DM you with more details 😄